src/style.css

```
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 15px;
+}
```

Сортуй правила згідно цього зразка - https://codeguide.academy/html-css.html#css-order.
Тоді твій код легше читати і підтримувати
```suggestion
.card {
  width: 200px;
  height: 403px;
  padding: 30px 14px 15px;
   
  display: flex;
  flex-direction: column;
  align-items: center;
  
  background: #fff;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
}
```

---

done